### ISSUE TITLE
What’s electronic tagging?

### ISSUE THREAD

**Issue by fdunnington** [criminaljusticehub/sprint-board#159]
https://www.criminaljusticehub.org.uk/task/whats-electronic-tagging/

**Comment by fdunnington** [comment]
The information on .gov website is quite different. Does this page need completely re-writing to match .gov info?

**Comment by legaleagleryan** [comment]
Yes, we could rewrite it based on the updated guidance. I have found some information which goes into a bit more detail about it which may help. 

https://www.gov.uk/government/publications/electronic-monitoring-in-the-criminal-justice-system/electronic-monitoring-in-the-criminal-justice-system

https://www.college.police.uk/research/crime-reduction-toolkit/electronic-tagging-sex-offences

https://www.sentencingcouncil.org.uk/pronouncement-cards/card/curfew-requirement/

https://prisonreformtrust.org.uk/adviceguide/home-detention-curfew-hdc/

**Comment by legaleagleryan** [comment]
Please see a recent response from the MOJ to a FOIA request relating to electronic tagging which may be useful at [WhatDoTheyKnow.com](https://www.whatdotheyknow.com/request/suppliers_electronic_monitoring#incoming-2694519)
